builder/parser/testfiles/continueIndent/Dockerfile

```diff
@@ -13,7 +13,7 @@ world
 RUN echo hello \
 goodbye\
 frog
-RUN echo hello  \  
```

Comment:
you might as well exclude all files under builder/parser/testfiles - just to be safe.  Not just from this PR but from your validation script too.